JinGine.WinForms/PointExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace JinGine.WinForms;
+
+internal static class PointExtensions
+{
+    internal static void CropX(ref this Point point, int min, int max) =>
+        point.X = Math.Max(min, Math.Min(max, point.X));
+
+    internal static void CropY(ref this Point point, int min, int max) =>
+        point.Y = Math.Max(min, Math.Min(max, point.Y));
+}
```

Comment:
restore Int32Extensions.Crop to replace this 2 methods and other calls that could be done in EditorTextViewer.EnsureVisibleCaret

Reply:
was wrong about EditorTextViewer.EnsureVisibleCaret(), but replace it anyway